infra/src/main/kotlin/fi/fta/geoviite/infra/inframodel/InfraModelConversion.kt

```diff
@@ -368,7 +369,7 @@ fun toGvtCantPoint(station: InfraModelCantStation): GeometryCantPoint {
             null -> LINEAR
             "biquadraticParabola" -> BIQUADRATIC_PARABOLA
             else -> throw InputValidationException(
-                message = "Unknown XML Cant transition type: ${formatForException(station.transitionType)}",
+                message = "Unknown XML Cant transition type: ${formatForException(station.transitionType as String)}",
```

Comment:
Miksi tämä castataan?

Reply:
tässä tuli muuten valitus `Smart cast to 'String' is impossible, because 'station.transitionType' is a property that has open or custom getter` ja ajattelin, että tässä voisi castata tai käyttää non-null asserted callia !!

Reply:
tuo muuttui siis samalla, kun lisäsin interfacen